src/server.h

```diff
@@ -271,6 +271,8 @@ class InferenceServer {
     enable_model_namespacing_ = e;
   }
 
+  void SetPeerAccessEnabled(const bool e) { enable_peer_access_ = e; }
```

Comment:
should match with naming above, either ``SetEnablePeerAccess` or `SetPeerAccess`